### PR TITLE
SUBMARINE-389. [WEB]Implement the home page in workbench with Angular

### PR DESCRIPTION
**Pull request by kevin85421**
### What is this PR for?
1. Implement the home page in the workbench with Angular.
2. The style configuration "workbenchContent" caused impacts on the whole workbench, but the configuration is only for the workspace component. In addition, the frontend of the Manager/User page is influenced. Due to this reason, the style configuration needs to be separated from the whole workbench to the workspace component only.

### What type of PR is it?
[Feature]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-389

### How should this be tested?
https://travis-ci.org/apache/submarine/builds/656336532?utm_medium=notification&utm_source=github_status
### Screenshots (if appropriate)
case1: >= 1600px
截圖 2020-02-29 上午12 45 34

case2: < 1600px
截圖 2020-02-29 上午12 48 46

workspace page
截圖 2020-02-29 上午12 47 02

Manager/User page
截圖 2020-02-29 上午12 50 29

### Questions:
* Does the licenses files need an update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kevin85421** [comment]
@jasoonn @liuxunorg Can you help me review the pull request? Thanks!

File: submarine-test/e2e/src/test/java/org/apache/submarine/integration/homeIT.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    changePageIndexButtons.get(3).click();
+    Assert.assertEquals( driver.findElements(By.xpath("//div[contains(text(), \"Submarine supports yarn 2.7.x 123\")]")).size(), 5);
+    changePageIndexButtons.get(3).click();
+    Assert.assertEquals( driver.findElements(By.xpath("//div[contains(text(), \"Submarine supports yarn 2.7.x 456\")]")).size(), 5);
+    changePageIndexButtons.get(2).click();
+    Assert.assertEquals( driver.findElements(By.xpath("//div[contains(text(), \"Submarine supports yarn 2.7.x 123\")]")).size(), 5);
+    changePageIndexButtons.get(2).click();
+    Assert.assertEquals( driver.findElements(By.xpath("//div[contains(text(), \"Submarine supports yarn 2.7.x\")]")).size(), 5);
+  }
+}
```
Comment (jasoonn) [inline submarine-test/e2e/src/test/java/org/apache/submarine/integration/homeIT.java]: @kevin85421 Thank for you contribution.
No newline at the end of the file.

Comment (kevin85421) [reply submarine-test/e2e/src/test/java/org/apache/submarine/integration/homeIT.java]: @jasoonn Thank you for your review. I have fixed the problem.

**Comment by kevin85421** [comment]
@liuxunorg Thank you for your review!
@jasoonn I have updated the pull request, and some changes are related to the workspace component. Can you help me review the new changes? Thanks a lot!

**Comment by jasoonn** [comment]
@kevin85421 Thank you for contribution.
They are great!

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**